src/bootstrap/src/core/builder/mod.rs

```diff
@@ -1386,7 +1386,7 @@ impl<'a> Builder<'a> {
         // Windows doesn't need dylib path munging because the dlls for the
         // compiler live next to the compiler and the system will find them
         // automatically.
```

Comment:
Nit: please also update comment above

Reply:
Well, cygwin is also Windows so I think it's OK...

Reply:
Hm, fair enough.